sfepy/terms/terms_elastic.py

```diff
@@ -636,3 +636,77 @@ def get_eval_shape(self, ts, mat0, mat1, parameter,
         out = CauchyStressTerm.get_eval_shape(self, mat0, parameter, mode,
                                               term_mode, diff_var, **kwargs)
         return out
+
+class NonsymetricElasticTerm(Term):
+    r"""
+    Elasticity term with non-symetric gradient. The indices of matrix
+    :math:`D_{ijkl}` are ordered as
+    :math:`[11, 12, 13, 21, 22, 23, 31, 32, 33]` in 3D and as
+    :math:`[11, 12, 21, 22]` in 2D.
+
+    :Definition:
+
+    .. math::
+        \int_{\Omega} \uul{D} \ul{u} : \ul{v}
+
```

Comment:
Typos are fixed.

I prefer NonsymElasticTerm for the class name.

I intend to use this term in the context of homogenization where the "super-set" ordering is not too convenient and might be confusing.